src/Microsoft.ML.Data/Transforms/SkipTakeFilter.cs

```diff
@@ -108,6 +108,18 @@ public static SkipTakeFilter Create(IHostEnvironment env, Arguments args, IDataV
             return new SkipTakeFilter(skip, take, env, input);
         }
 
+        /// <summary>
+        /// A helper method to create 'SkipFilter' for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">>Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="count">Number of rows to skip</param>
+        /// <returns></returns>
+        public static SkipTakeFilter CreateSkipFilter(IHostEnvironment env, IDataView input, long count = Arguments.DefaultSkip)
```

Comment:
This is another case where having static classes like `SkipFilter` and `TakeFilter` both with `Create` methods might make for a more sensible API. So instead of usage being

```csharp
SkipTakeFilter.CreateSkipFilter
SkipTakeFilter CreateTakeFilter
```

it might instead look like

```csharp
SkipFilter.Create
TakeFilter.Create
```